registry/api/v2/descriptors.go

```diff
@@ -374,7 +346,7 @@ var routeDescriptors = []RouteDescriptor{
 									ContentType: "application/json; charset=utf-8",
 									Format:      errorsBody,
 								},
-								ErrorCodes: []ErrorCode{
+								ErrorCodes: []errcode.ErrorCode{
```

Comment:
if I'm following you, you're saying we might be able to get rid of ResponseDescriptor.StatusCode and just use the code from the Error, right? If so, I'm not sure we can.  For the error cases I agree, that's duplicate info.  But in the non-error cases it appears that not all of the ResponseDescriptor's are 200's, some are 202's - and assuming we want to keep that distinction we'd need to continue to have something for that info.  

Now, I could make it so that if ResponseDescriptor.ErrorCodes is not nil then grab the http status code from the first ErrorCode; and if ErrorCodes is nil and ResponseDescriptor.StatusCode is zero then default to 200.  But allow for ResponseDescriptor.StatusCode to also be set to override the default 200 (e.g. 202).

Let me know what you think.